lua/lazyvim/plugins/extras/test/core.lua

```diff
@@ -0,0 +1,51 @@
+return {
+  {
+    "nvim-neotest/neotest",
+    dependencies = {
+      -- which key integration
+      {
```

Comment:
nitpick, but you can remove this as a dep and just add it separately avove the neotest spec

Reply:
Done. Added this fancy new `optional` keyword too 😄